lib/mayaUsd/resources/scripts/mayaUsdOptions.py

```diff
@@ -41,10 +56,28 @@ def convertOptionsDictToText(optionsDict):
     """
     Converts options to text with column-separated key/value pairs.
     """
-    optionsList = ['%s=%s' % (name, value) for name, value in optionsDict.items()]
+    optionsList = ['%s=%s' % (name, _convertValueToText(value)) for name, value in optionsDict.items()]
     return ';'.join(optionsList)
 
 
+def _convertValueToText(value):
+    # Unfortunately, for historical reasons, text lists have used
+    # comma-separated values, while floating-point used space-separated values.
+    if isinstance(value,list):
+        # Note: empty list must create empty text.
+        if not len(value):
```

Comment:
What you have is okay, but you can also use 
```suggestion
        if not value:
```

Reply:
True! OTOH, the code as written makes clear that its the length that we're diverging on, not any other falsy reason, like the variable being None. If there were other changes needed I'd do it, but I don't think it is worth a new batch of pre-flight.

Reply:
OTOH, github let us commit the change right here, so if you feel like it, you can commit your suggestion before merging.